test/stubs/is_big_endian.c

```diff
@@ -0,0 +1,5 @@
+#include "util.h"
+
+static bool IS_LITTLE_ENDIAN = false;
+
+bool is_big_endian(void) { return IS_LITTLE_ENDIAN; }
```

Comment:
Haha, I know it's a little thing that's in progress, but let's go ahead and make this not so obviously wrong before merging.

Reply:
Oops! I will remove that file it crept in there.